.github/workflows/test.yaml

```
@@ -122,6 +122,9 @@ jobs:
           # Just Python 3.9 with default settings.
           - python-version: '3.9'
 
+          # Just Python 3.11 with default settings.
+          - python-version: '3.11'
+
           # Newest macOS and newest Python supported versions.
           - python-version: '3.10'
```

My suggestion was to use 3.11 here 

```suggestion
          - python-version: '3.11'
```

---

Ah, makes sense.  Updated.